packages/plugin-core/src/web/utils/WSUtils.ts

```diff
@@ -14,19 +22,19 @@ export class WSUtilsWeb {
     @inject("vaults") private vaults: DVault[]
   ) {}
 
-  private getVaultFromDocument(document: vscode.TextDocument) {
+  getVaultFromDocument(document: vscode.TextDocument) {
     const txtPath = document.uri.fsPath;
     const vault = VaultUtils.getVaultByFilePath({
-      wsRoot: this.wsRoot.fsPath,
+      wsRoot: normalizeUnixPath(this.wsRoot.fsPath),
       vaults: this.vaults,
-      fsPath: txtPath,
+      fsPath: normalizeUnixPath(txtPath),
```

Comment:
used normailzed path values here. eventually we compare them normalized anyway in `getVaultByDirPath` and also because I couldn't work around a `Utils` alternative for `path.relative` [here](https://github.com/dendronhq/dendron/blob/master/packages/common-all/src/vault.ts#L234)